package.json

```diff
@@ -96,14 +96,20 @@
             "markdownDescription": "The path of the folder containing the Swift executables. The default is to look in **$PATH**.",
             "order": 1
           },
+          "swift.runtimePath": {
+            "type": "string",
+            "default": "",
+            "description": "The path of the folder containing the Swift runtime libraries. This will only affect runtime behavior when libraries are not discovered by RPath.",
```

Comment:
Wording of this can be improved. Should we use something similar to the sdk description